docker/venice-controller/single-dc-configs/controller.properties

```diff
@@ -22,7 +22,7 @@ enable.hybrid.push.ssl.whitelist=false
 default.partition.size=100
 min.active.replica=1
 kafka.replication.factor=1
-child.cluster.allowlist=false
+child.cluster.allowlist=
```

Comment:
This will end up in `Utils.parseCommaSeparatedStringToList` where this is the updated code:
```
if (StringUtils.isEmpty(rawString)) {
  return Collections.emptyList();
}
```

Also, this config is not used in child-controllers. It is only ever used in parent controllers. The reason I had to include this change is that there is a new code that checks if this config is set and infers it as a multi-region mode. Also, the value of `false` is absurd for this field, and hence, also incorrect.

Ideally, this config shouldn't be set, but I still wanted this left behind to reduce the number of changes, and also show that it is actually backwards-compatible if this was configured correctly in the first place. I think backwards-compatibility for misconfigured stuff can be a tricky topic and I think it is okay to break compatibility in such cases.

Reply:
I see.. agree false was anyways a bad default in the first place. Sounds good 👍